src/main/kotlin/net/atos/zac/shared/model/SorteerRichting.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos, 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.shared.model
+
+import net.atos.zac.shared.model.SorteerRichting.ASCENDING
+import net.atos.zac.shared.model.SorteerRichting.DESCENDING
+
+enum class SorteerRichting(val value: String) {
+    /** Oplopend  */
+    ASCENDING("asc"),
+
+    /** Aflopend  */
+    DESCENDING("desc")
+}
+
+fun fromValue(waarde: String?): SorteerRichting =
```

Comment:
```suggestion
fun fromValue(waarde: String): SorteerRichting =
```

Surely?

Reply:
These come from rest query params, so we **can** have `null`